Update china-get-started-service-availability.md

josefree opened the pull request:
Marketplace and Azure Monitor have GAed on Mooncake. Please check www.azure.cn for confirmation.

azurecla commented:
Hi __@AlekseiPolkovnikov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@AlekseiPolkovnikov : Thanks for your contribution to the Azure documentation! The author, @georgewallace, has been notified to review your proposed change.

georgewallace commented:
@AlekseiPolkovnikov I submitted a PR to your fork.  I removed the verbose message from the output and left the text in the step alone, the verbose output should not have been there in the first place. Great catch.

AlekseiPolkovnikov commented:
Agree, no need in verbose output in this place.

azurecla commented:
Hi __@AlekseiPolkovnikov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

AlekseiPolkovnikov commented:
Re-opened, was closed by accident.

josefree committed:
Update china-get-started-service-availability.md

Marketplace and Azure Monitor have GAed on Mooncake. Please check www.azure.cn for confirmation.
